content/tutorials/content-types.md

```diff
@@ -10,32 +10,237 @@ A Content Type Definition (CTD) is an XML-format configuration file for content
+
+![Add new contract form](./img/newcontract2.png "Add new contract form with some hidden fields")
+
+In this case adding this three fields to the content type does not mean you've created totally new fields. These fields will be handled as updated ones which can be done based on the fact that the name and type pairs of a field should be unique helping the type system identify fields to update.
+
+<note severity="info">Learn more about field visibility in the <a href="/concepts/fields/10-datetime">concepts</a></note>
 
 # Add a new field
 
+Of course you can add new fields to any type, but only by adding them with a unique name. If a field with a same name exists in the system there are two possible outcomes: if a field with a same name and type exists in the schema, the modifications will override the existing field configuration, if a field with the same name exists but with a different type, you will not be able to save the CTD.
+
+To try out new field creation, add the following field definition before the closing `</Fields>` tag.
+
+```xml
+<Field name="IsSigned" type="Boolean">
+  <DisplayName>Signed</DisplayName>
+</Field>
+```
+
+As you see the new field is a simple boolean field to store if the contract has already been signed.
+
+![Add new contract form](./img/newcontract3.png "Add new contract form with a new field")
+
 # Modify fields
 
-- no type change
+You may want to change a field and its configuration for many reasons: to change order of the fields, or their visibility and so on. Every aspect of a field could be modified except one, the type. Fields have various configurations and settings per type. To learn more about a specific field type's possible configurations check the [concept docs](https://docs.sensenet.com/concepts/fields).
 
-# Working with field visibility
+To try out modification add a default value to the *Contract value*. Update the field definition with the following configuration:
+
+```xml
+<Field name="ContractValue" type="Number">
+    <DisplayName>Contract value</DisplayName>
+    <Description>Value of the contract</Description>
+    <Configuration>
+      <Digits>2</Digits>
+      <DefaultValue>100</DefaultValue>
+    </Configuration>
+  </Field>
+```
+
+Now when you create a new contract the value field will be automatically filled with the default value.
+
+![Add new contract form](./img/newcontract4.png "Add new contract form with a new field")
 
 # Delete a field
 
+To actually delete a field, we need to delete it from the CTD where it is basically defined. So for example if we want to delete `EnableLifespan` field system wide, we have to remove it from the `GenericContent` type. It is easier if we are working with custom content types, because the non-inherited custom fields could be easily removed from the custom content types.
```

Comment:
```suggestion
To actually delete a field, we need to delete it from the CTD where it is basically defined. So for example if we want to delete `EnableLifespan` field system-wide, we have to remove it from the `GenericContent` type. It is easier if we are working with custom content types, because the non-inherited custom fields could be easily removed from these.
```